docker/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - MYSQL_DATABASE=domjudge
     ports:
       - 13306:3306
-    command: --max-connections=1000 --max-allowed-packet=512M
+    command: --max-connections=1000 --max-allowed-packet=512M --innodb_snapshot_isolation=OFF
```

Comment:
This includes a fix for an issue we had during NWERC 2024, which was occurring due to a new default variable change in the newest version of MariaDB, which locked certain transactions, therefore sometimes causing issues for DOMjudge and the judgehosts.